services/sn_analysis_service/app.py

```diff
@@ -112,55 +138,81 @@ async def run_sn_model(dd):
             model.param_names,
             bounds=bounds,
         )
-        log(f"Number of χ² function calls: {result.ncall}")
-        log(f"Number of degrees of freedom in fit: {result.ndof}")
-        log(f"χ² value at minimum: {result.chisq}")
-        log(f"model parameters: {result.param_names}")
-        log(f"best-fit values: {result.parameters}")
-        log(f"The result contains the following attributes:\n {result.keys()}")
-        log(f'{result["success"]=}')
-
-        if result["success"]:
-
-            with tempfile.NamedTemporaryFile(
-                suffix=".png", prefix="snplot_", dir="/tmp/"
-            ) as f:
-                _ = sncosmo.plot_lc(
-                    data,
-                    model=fitted_model,
-                    errors=result.errors,
-                    model_label=source,
-                    zp=23.9,
-                    figtext=data_dict["resource_id"],
-                    fname=f.name,
-                )
-                f.seek(0)
-                plot_data = io.BytesIO(f.read())
-                f.close()
-                os.remove(f.name)
-                log('made lightcurve plot')
+
+        if result.success:
+            f = tempfile.NamedTemporaryFile(
+                suffix=".png", prefix="snplot_", dir="/tmp/", delete=False
```

Comment:
Cannot guarantee that we can write to /tmp. `./` tmp may be okay, but rather just let Python figure out an appropriate dir.